### DISCUSSION TITLE
GET /_next/image randomly start returning HTTP 500  on Azure App Service static website

### DISCUSSION THREAD

**Discussion by alexdefaro** [vercel/next.js#38627] [Help] [answered]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

NextJS 12.0.7
NodeJS Node 14 LTS 
Static Website 
Azure App Service

### What browser are you using? (if relevant)

Any browser 

### How are you deploying your application? (if relevant)

NextJS 12.0.7 SSG and Azure App services  

### Describe the Bug

We have a static website generated using NextJS and for the images we use NextJS Image component. What happens is that images on the website  randomly starts returning HTTP 500 after a while.  The HTTP 500 returns for random images in the website and at random points in time. Some time's it takes days and some just minutes after we deploy the website. When it happens we have to restart the App Service so the error disappears and it starts working again. Only to come back again. 

GET /_next/image returns HTTP 500

### Expected Behavior

Expect the images to continue loading without any errors.

### Link to reproduction

we don't have it.

### To Reproduce

Since this is a random issue... There is no way to reproduce it.

**Comment by GabenGar** [comment]
Pretty sure you have to write your [custom loader](https://nextjs.org/docs/api-reference/next/image#loader) to use `<Image>` for static exports.

**Comment by alexdefaro** [comment, marked as answer]
We updated NextJS to the last version and it solved the issue.
